src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -835,6 +845,41 @@ pub enum RoundMode {
 }
 
 impl Decimal {
+    /// abs the Decimal into a new Decimal.
+    #[inline]
+    pub fn abs(mut self) -> Res<Decimal> {
+        self.negative = false;
+        Res::Ok(self)
+    }
+
+    /// ceil the Decimal into a new Decimal.
+    pub fn ceil(&self) -> Res<Decimal> {
```

Comment:
Any test to cover this?

Reply:
Yes, but they are in `coprocessor/dag/expr/math.rs`.

Reply:
They are different. The unit test for this function should focus on its functionality, while tests in math.rs is used to validate if the eval is implemented correctly.

Reply:
fixed.